### PR TITLE
negar/pa_input

### PR DESCRIPTION
**Pull request by negarn**
add PA input field

### CODE REVIEW

File: src/javascript/app/pages/cashier/payment_agent_withdraw.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+const refreshDropdown      = require('@binary-com/binary-style').selectDropdown;
```
Comment (cakasuma) [inline src/javascript/app/pages/cashier/payment_agent_withdraw.js]: do we need to alphabetize import order for binary static?

File: src/sass/app/websocket_pages.scss [hunk 2]
```diff
@@ -300,3 +300,65 @@
...
+                    text-align: center;
+                }
+            }
+            .hint {
+                width: min-content;
+                line-height: 1.4;
+            }
+        }
+    }
+    .select-dropdown, #txtAgents, #txtAmount, .row-agent .hint, .row-agent > div:nth-child(2) {
```
Comment (cakasuma) [inline src/sass/app/websocket_pages.scss]: imo, its better to break line here?

File: src/sass/app/websocket_pages.scss [hunk 3]
```diff
@@ -300,3 +300,65 @@
...
     }
 }
+
+#paymentagent_withdrawal {
+    .wrapper-row-agent {
+        label {
+            padding-bottom: 10px;
+        }
+        .row-agent {
+            width: fit-content;
```
Comment (cakasuma) [inline src/sass/app/websocket_pages.scss]: if im not mistaken, fit-content doesn't work on edge/ie

File: src/javascript/app/pages/cashier/payment_agent_withdraw.js (hunk 1 above)
Comment (negarn) [reply src/javascript/app/pages/cashier/payment_agent_withdraw.js]: it's based on path

File: src/sass/app/websocket_pages.scss (hunk 2 above)
Comment (negarn) [reply src/sass/app/websocket_pages.scss]: @cakasuma eslint gives error on multiline

**Pull request merged**